### PR TITLE
Support current Python versions only. 

### PR DESCRIPTION
**Pull request by sallner**
If this PR gets merged, it would be nice to have a release of the package.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,15 +4,26 @@ sudo : false
...
+    - python: "3.7"
+      dist: xenial
+      sudo: true
+      env: TOXENV=py37
+  include:
+    - python: "pypy"
+      env: TOXENV=pypy
+  include:
+    - python: "pypy3"
+      env: TOXENV=pypy3
```
Comment (tseaver) [inline .travis.yml]: Is there a reason for the more complicated `matrix:` dance here, versus just including all the supported Python versions in `env:`?

Comment (sallner) [reply .travis.yml]: I tried it before in the first commit, but there seems a problem with `travis_retry tox` in this combination, see travis log. One alternative is to change the setup more to the style currently used in the zopefoundation projects like https://github.com/zopefoundation/z3c.flashmessage/pull/2.

What do you prefer?

**Review by tseaver — APPROVED** [review]

**Comment by tseaver** [comment]
Release tagged, pushed to PyPI: https://pypi.org/project/repoze.profile/2.3/

**Comment by sallner** [comment]
Thanks for the quick release!

**Pull request merged**